gnome-shell/src/gnome-shell-sass/_common.scss

```diff
@@ -162,6 +162,7 @@ StScrollBar {
   -slider-height: 0.2em; //increase the height because of the border removal
   -slider-background-color:  mix($borders_color, $bg_color, 50%);
   -slider-border-color: $borders_color; //trough border color
+  -slider-border-width: 1px;
```

Comment:
I think they've add the corresponding JS for this property after bionic and this might not work 🤔